<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>761</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>725</width>
      <height>725</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>725</width>
      <height>725</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="cmethodCBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>240</width>
      <height>25</height>
     </rect>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtBottom</enum>
    </property>
    <item>
     <property name="text">
      <string>Каноническое уравн.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Параметрическое ур.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Брезенхем</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Средней точки</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Библиотечный</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="colorLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="fgColorBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>440</y>
      <width>170</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Основной цвет</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bgToFgBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>240</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Цвет фона -&gt; основной</string>
    </property>
   </widget>
   <widget class="QLabel" name="bgColorLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bgColorBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>470</y>
      <width>170</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Фоновый цвет</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>240</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить экран</string>
    </property>
   </widget>
   <widget class="QPushButton" name="defaultFgBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>240</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Базовый основной цвет</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>240</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Статистика</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>251</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Круг</string>
     </attribute>
     <widget class="QPushButton" name="circleBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>240</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Построить окружность</string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>241</width>
        <height>110</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Xc</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Yc</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>R</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="xcEnt">
         <property name="maximum">
          <number>720</number>
         </property>
         <property name="value">
          <number>360</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="ycEnt">
         <property name="maximum">
          <number>720</number>
         </property>
         <property name="value">
          <number>360</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="rEnt">
         <property name="maximum">
          <number>360</number>
         </property>
         <property name="singleStep">
          <number>0</number>
         </property>
         <property name="value">
          <number>180</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Rs</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Re</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>260</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>N</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>230</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>dR</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="rsEnt">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>210</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>720</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="reEnt">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>210</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>720</number>
      </property>
      <property name="value">
       <number>360</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="drEnt">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>210</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="nEnt">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>210</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>720</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="concCirclesBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>290</y>
        <width>240</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Построить окружности</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Эллипс</string>
     </attribute>
     <widget class="QPushButton" name="ellipseBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>240</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Построить эллипс</string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>241</width>
        <height>148</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Xc</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="elxcEnt">
         <property name="maximum">
          <number>720</number>
         </property>
         <property name="value">
          <number>360</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Yc</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="elycEnt">
         <property name="maximum">
          <number>720</number>
         </property>
         <property name="value">
          <number>360</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="aEnt">
         <property name="maximum">
          <number>360</number>
         </property>
         <property name="singleStep">
          <number>0</number>
         </property>
         <property name="value">
          <number>180</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="bEnt">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>360</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>B0</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="elNEnt">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>210</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="value">
       <number>25</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="elDrEnt">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>210</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>720</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>A0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="concEllipseBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>330</y>
        <width>240</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Построить эллипсы</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="a0Ent">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>210</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>720</number>
      </property>
      <property name="value">
       <number>75</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="b0Ent">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>210</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>720</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>dR</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>N</string>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>tabWidget</zorder>
   <zorder>graphicsView</zorder>
   <zorder>cmethodCBox</zorder>
   <zorder>colorLbl</zorder>
   <zorder>fgColorBtn</zorder>
   <zorder>bgToFgBtn</zorder>
   <zorder>bgColorLbl</zorder>
   <zorder>bgColorBtn</zorder>
   <zorder>clearBtn</zorder>
   <zorder>defaultFgBtn</zorder>
   <zorder>pushButton</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
